fboss/agent/test/TestUtils.h

```diff
@@ -267,7 +267,7 @@ class TxPacketMatcher
       folly::StringPiece name,
       TxMatchFn&& fn);
 
-  bool MatchAndExplain(
+  virtual bool MatchAndExplain(
```

Comment:
Did you mean to check in this change?  I thought you said it didn't help with the compilation issue?